### DISCUSSION TITLE
x3 Playback speed in v19.02.1-3.0.1

### DISCUSSION THREAD

**Discussion by MajesNth** [qnblackcat/uYouPlus#1294] [Q&A]
I just got the latest uYouPlus IPA v19.02.1-3.0.1 and I noticed that the maximum playback speed is now x2 instead of the old x3. Is it possible to change it back to greater than 2.00 in the newer version as well? I found that option to be very handy when fast forwarding

**Comment by qnblackcat** [comment]
For now you can play at 3x by using uYou Video playback controls
